app/routines/newflow/create_or_update_salesforce_lead.rb

```diff
@@ -121,31 +121,19 @@ def exec(user:)
             event_data: { lead_id: lead.id.to_s }
           )
         else
+          if lead.errors.messages.inspect.include? == 'INSUFFICIENT_ACCESS_ON_CROSS_REFERENCE_ENTITY'
+            Sentry.capture_message("Invalid school (#{user.school.salesforce_id}) for user (#{user.id})")
```

Comment:
It's not clear to me that's what the message means but if you are sure from what you've seen in SF go for it.

Reply:
This took a bit of command line testing to figure out what SF is spitting out as an error - but yeah, this is the one's that need catching with the ID of that school so it can at least be manually cleaned up (maybe this can move to the admin or some automated process in the future).